sdk/search/azure-search-documents/src/test/java/com/azure/search/documents/indexes/IndexManagementTests.java

```diff
@@ -55,6 +57,7 @@
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
+@Execution(ExecutionMode.SAME_THREAD)
```

Comment:
Don't need this as live testing defaults to using `SAME_THREAD`.

Reply:
Okay, sounds good